Content.Shared/Roles/DepartmentPrototype.cs

```diff
@@ -9,7 +9,13 @@ public sealed partial class DepartmentPrototype : IPrototype
     public string ID { get; } = string.Empty;
 
     /// <summary>
-    /// A description string to display in the character menu as an explanation of the department's function.
+    /// The name LocId of the department that will be displayed in the various menus.
+    /// </summary>
+    [DataField(required: true)]
+    public LocId Name = string.Empty;
+
+    /// <summary>
+    /// A description LocId to display in the character menu as an explanation of the department's function.
     /// </summary>
     [DataField(required: true)]
     public string Description = string.Empty;
```

Comment:
The summary says that the Description is a LocId but it is still just a string

Reply:
oh noooo

Reply:
fixed